src/ikernel.ts

```diff
@@ -15,6 +15,10 @@ import {
 } from 'jupyter-js-utils';
 
 
+export
+type hist_access_type = 'range' | 'tail' | 'search';
```

Comment:
The underscore case was to match the spec, but the variable name doesn't need to.